src/js/mobile-menu.js

```diff
@@ -0,0 +1,65 @@
+/* global $ */
+(function () {
+
+  let menuSelector;
+  let menuTriggerSelector;
+  let $menuTrigger;
+  let menuTriggerClass;
+  let $menu;
+  let menuSlideoutClass;
+  let menuCloseSelector;
+  let $bodyOverlay;
+  let bodyOverlayClass;
+
+
+  function closeMenu() {
+    $menu.removeClass(menuSlideoutClass);
+    $menuTrigger.removeClass(menuTriggerClass);
+    $bodyOverlay.removeClass(bodyOverlayClass);
+  }
+
+  function toggleMenu() {
+    $menu.toggleClass(menuSlideoutClass);
+    $menuTrigger.toggleClass(menuTriggerClass);
+    $bodyOverlay.toggleClass(bodyOverlayClass);
+  }
+
+  function closeMobileMenuHandler() {
+    $(document).click((e) => {
+      const $target = $(e.target);
+
+      // Hide the mobile menu when anything else is clicked
+      if (!$target.closest(menuSelector).length && !$target.closest(menuTriggerSelector).length) {
+        if ($menu.width() > 200) {
+          closeMenu();
+        }
+      }
+    });
+  }
+
+  function closeMobileMenuIconHandler() {
+    $menu.find(menuCloseSelector).click(closeMenu);
```

Comment:
Rather than calling the find method each time the icon is clicked, it would be more efficient to save the menuCloseSelector object to a variable.

Reply:
Adjusted in 3c037e3